pyproject.toml

```diff
@@ -230,11 +230,7 @@ xfail_strict = true
 # This is for regtests. All unhandled warnings become exceptions for
 # regular CI in tox.ini file.
 filterwarnings = [
-    "error::ResourceWarning",
-    "error::DeprecationWarning",
-    "error::stdatamodels.exceptions.ValidationWarning",
-    "error:Invalid interval",
-    "ignore:Models in math_functions:astropy.utils.exceptions.AstropyUserWarning",
+    "error"
```

Comment:
Just to ensure I understand, the entry "error" in `filterwarnings` here means that all uncaptured warnings will return as errors?